src/layout/organize/JourneyInstanceLayout.tsx

```diff
@@ -36,16 +36,11 @@ const JourneyInstanceLayout: React.FunctionComponent = ({ children }) => {
 
   const ellipsisMenu: ZetkinEllipsisMenuProps['items'] = [];
 
-  ellipsisMenu.push({
-    id: 'convert-journey',
-    label: intl.formatMessage({
-      id: 'pages.organizeJourneyInstance.ellipsisMenu.convert',
-    }),
-    startIcon: <Forward color="secondary" />,
-    subMenuItems: journeys
+  const subMenuItems =
+    journeys
       ?.filter((journey) => journey.id.toString() !== journeyId)
       .map((journey) => ({
-        id: `convert-journey-submenu-${journey.singular_label}`,
+        id: `convertTo-${journey.id}`,
```

Comment:
I just like this better. `EllipsisMenu` (or actually `EllipsisSubMenu`) is already appended, so we don't need a long prefix to make this unique and the ID is easier to work with than the string label.